thingsboard.github.io/docker/README.md

```diff
@@ -0,0 +1,57 @@
+# thingsboard.github.io
+
+Based on [ruby:3.1](https://hub.docker.com/_/ruby). For detailed info see origin repository.
+
+## Deployment of the site in docker
+
+The commands below set up the environment for running GitHub pages in the docker. You do not need to install additional dependencies and packages, everything is already built into the docker image.
+
+If you do not have docker installed, you need to install it. You can do this by following the installation instructions: [Docker Engine installation overview](https://docs.docker.com/engine/install/)
+
+Put the site contents and the docker-compose file in the same directory:
+
+```bash
+mkdir thingsboard_github_io-docker && cd $_
+git clone https://github.com/thingsboard/thingsboard.github.io.git
+
+cat <<EOT | sudo tee docker-compose.yml
+version: '3.1'
+services:
+  tb_github_io:
+    container_name: thingsboard_github_io
+    restart: always
+    image: "THINGSBOARD_GITHUB_IO_IMAGE"
+    ports:
+      - "4000:4000"
+    volumes:
+      - ./thingsboard.github.io:/thingsboard.github.io
+EOT
+```
+
+Execute this command to add webrick for jekyll:
+```
+docker-compose run --no-deps --rm thingsboard_github_io bundle add webrick
+```
+
+To start the docker container with docker-compose, run the command:
+
+```bash
+docker-compose up
```

Comment:
Avoid docker-compose to make sure one can launch website with single line command without downloading and editing any files.

Reply:
Added documentation to start the docker container with one command.